malariagen_data/anoph/to_plink.py

```diff
@@ -0,0 +1,164 @@
+from typing import Optional
+
+import allel  # type: ignore
+import numpy as np
+import os
+import bed_reader
+
+from ..util import dask_compress_dataset
+from .snp_data import AnophelesSnpData
+from . import base_params
+from . import pca_params
+
+
+class PlinkConverter(
+    AnophelesSnpData,
+):
+    def __init__(
+        self,
+        **kwargs,
+    ):
+        # N.B., this class is designed to work cooperatively, and
+        # so it's important that any remaining parameters are passed
+        # to the superclass constructor.
+        super().__init__(**kwargs)
+
+    def _create_plink_outfile(
+        self,
+        *,
+        output_dir,
+        region,
+        n_snps,
+        min_minor_ac,
+        thin_offset,
+        max_missing_an,
+    ):
+        return f"{output_dir}/{region}.{n_snps}.{min_minor_ac}.{thin_offset}.{max_missing_an}"
```

Comment:
```suggestion
        return f"{output_dir}/{region}.{n_snps}.{min_minor_ac}.{max_missing_an}.{thin_offset}"
```

This is a very small suggestion, but I wonder if it's a little more intuitive if the `thin_offset` component of the path name comes at the end. This is because in the typical use case for an analysis like ADMIXTURE the user might want to create multiple replicate datasets with different sets of SNPs, and the way to do that here is to vary the `thin_offset`. I.e., a user might call the same function with `thin_offset` values of 0, 1, 2, etc. If this is the last path component then perhaps easier to see these are replicates?